requirements.txt

```diff
@@ -10,12 +13,23 @@ itsdangerous==1.1.0
 Jinja2==2.11.1
 Mako==1.1.2
 MarkupSafe==1.1.1
+marshmallow==3.5.1
+mccabe==0.6.1
+mirakuru==2.2.0
 more-itertools==8.2.0
 packaging==20.3
 pluggy==0.13.1
+port-for==0.4
+psutil==5.7.0
 psycopg2==2.8.4
 py==1.8.1
+pycodestyle==2.5.0
+pyflakes==2.1.1
 pyparsing==2.4.6
+pytest==5.4.1
+pytest-flask-sqlalchemy==1.0.2
```

Comment:
Pytest and other related requirements should be in requirements-dev.txt. When we go to production, we won't import those requirements.